### PR TITLE
refactor(scrollbar): clearing remaining components

### PR DESCRIPTION
**Pull request by ThomBos**
**What this PR does** 📖
Clears last few scrollbar components and clear all related code
**Which issue(s) this PR fixes** 🔨
AP-1730

**Special notes for reviewers** 🗒️
This PR depends on #3284  
**Additional comments** 🎤

### CODE REVIEW

**Comment by stavares843** [comment]
https://github.com/Satellite-im/Core-PWA/pull/3284 merged 🎉

**Review by phillsatellite — APPROVED** [review]

File: components/views/navigation/slimbar/Slimbar.html [hunk 1]
```diff
@@ -1,51 +1,26 @@
...
-              :name="server.name"
-            />
-          </div>
-        </UiComingSoon>
-      </div>
-    </UiScroll>
-    <div id="list-container" v-else>
+
+    <UiHorizontalRule v-if="friendsWithUnreadMessages.length" />
+    <UiSimpleScroll scrollMode="vertical" scrollShow="scroll" v-else>
```
Comment (molimauro) [inline components/views/navigation/slimbar/Slimbar.html:23]: line 22 should go under the div in line 23

**Review by molimauro — CHANGES_REQUESTED** [review]
* minor
* remove uiscroll component in `components/interactables/UserPicker/UserPicker.html`
* remove uiscroll component

File: components/views/navigation/slimbar/Slimbar.html (hunk 1 above)
Comment (ThomBos) [reply components/views/navigation/slimbar/Slimbar.html:23]: should we scroll also the horizontal line?

Comment (molimauro) [reply components/views/navigation/slimbar/Slimbar.html:23]: without this when we have the user circles (above communities) nothing will appear on screen:
image

Comment (ThomBos) [reply components/views/navigation/slimbar/Slimbar.html:23]: is it supposed to disappear when we don't have any chat?

Comment (molimauro) [reply components/views/navigation/slimbar/Slimbar.html:23]: nope, but in this case we have both and nothing appears

Comment (ThomBos) [reply components/views/navigation/slimbar/Slimbar.html:23]: ok, so the real issue is the if

**Comment by molimauro** [comment]
I have these overlapping:
image

Also the horizontal div is not centered:

image

**Comment by phillsatellite** [comment]
Tested: Mauro's bugs were fixed now but it looks like the changes brought up a new bug. Seems the pages within Settings are all being crunched to the left side of the screen
IMG_5605

**Comment by ThomBos** [comment]
> Tested: Mauro's bugs were fixed now but it looks like the changes brought up a new bug. Seems the pages within Settings are all being crunched to the left side of the screen !

....it looks pretty bad... can you send also a pic of each tab inside the modal please?

**Comment by ThomBos** [comment]
Fixed this issue, but it's necessary check every scrollbar now

File: components/ui/SimpleScroll/SimpleScroll.less [hunk 2]
```diff
@@ -5,7 +5,7 @@
   .wrapper {
     pointer-events: auto;
     width: @full;
-    height: @full;
+    height: fit-content;
```
Comment (molimauro) [inline components/ui/SimpleScroll/SimpleScroll.less:8]: Does this work on Firefox? https://caniuse.com/?search=fit-content

File: components/views/navigation/slimbar/Slimbar.less [hunk 3]
```diff
@@ -2,14 +2,13 @@
   height: @full;
   width: @slimbar-size;
   min-width: @slimbar-size;
-  display: inline-flex;
+  display: flex;
+  flex-direction: column;
   justify-content: flex-start;
   pointer-events: none;
-  padding-left: 0.6rem;
+  padding-left: 0.5rem;
```
Comment (molimauro) [inline components/views/navigation/slimbar/Slimbar.less:9]: @light-spacing

File: components/views/navigation/slimbar/Slimbar.less [hunk 4]
```diff
@@ -27,16 +26,15 @@
...
+  .circle {
+    margin: 0.25rem 0;
+    &:extend(.first-layer);
   }
 
   .unreads {
     position: relative;
     .unread {
-      margin: 0.5rem 0;
+      padding: 0.25rem 0;
```
Comment (molimauro) [inline components/views/navigation/slimbar/Slimbar.less:37]: @xlight-spacing

File: components/views/navigation/slimbar/Slimbar.less [hunk 5]
```diff
@@ -54,6 +52,16 @@
     overflow: hidden;
     display: flex;
   }
+  #list-container-vertical {
+    height: fit-content;
+    max-height: calc(@full - 11rem);
```
Comment (molimauro) [inline components/views/navigation/slimbar/Slimbar.less:57]: Can you explain this calc on a comment? :)

Comment (ThomBos) [reply components/views/navigation/slimbar/Slimbar.less:57]: 11 rem is settings and new community buttons' total height, in order to push those buttons out of the screen i had to leave that margin. maybe it's better creating a constant for that

File: components/views/navigation/slimbar/Slimbar.less [hunk 6]
```diff
@@ -54,6 +52,16 @@
...
+    pointer-events: auto;
+    box-sizing: border-box;
+
+    #simple-scrollbar {
+      max-height: calc(@full - 3rem);
```
[the remark this answers is not in the document]
Comment (ThomBos) [reply components/views/navigation/slimbar/Slimbar.less:62]: this 3 rems are for an inner padding/margin given by a divider

File: components/ui/SimpleScroll/SimpleScroll.less (hunk 2 above)
Comment (ThomBos) [reply components/ui/SimpleScroll/SimpleScroll.less:8]: searching for another solution...

**Comment by molimauro** [comment]
@ThomBos if you modify the code put the Missing review label thanks :)

**Review by molimauro — APPROVED** [review]

**Comment by ThomBos** [comment]
> @ThomBos if you modify the code put the Missing review label thanks :)

my bad 😬

**Review by stavares843 — APPROVED** [review]

**Pull request merged**